ubi9-openjdk-17-runtime.yaml

```diff
@@ -41,7 +41,7 @@ modules:
   install:
   - name: jboss.container.util.pkg-update
   - name: jboss.container.openjdk.jre
-    version: "3.8.17"
+    version: "17"
```

Comment:
Makes sense. Only the maven modules (not installed for runtime), have version scheme `<maven-version>.<jdk-version>`. Nonsensical for module `jboss.container.openjdk.jre`. Thumbs up.